ledfx/effects/spotlight.py

```
@@ -0,0 +1,460 @@
+        return random.randrange(self.pixel_count)
+
+    def _spawn_spot(self, now):
+        """Create a new spotlight and cap list size to max_active_spots."""
+        overflow = self.spot_centers.size - self.max_active_spots + 1
+        if overflow > 0:
+            self._drop_oldest_spots(overflow)
+
+        color_anchor = 0.0
+        if self.use_gradient:
+            color_anchor = (
+                self.gradient_phase + random.random() * self.spot_color_span
```

Ruff (0.15.10)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)